app/src/androidTest/java/org/mozilla/fenix/helpers/idlingresource/NetworkDisconnectedIdlingResource.kt

```diff
@@ -0,0 +1,32 @@
+package org.mozilla.fenix.helpers.idlingresource
+
+import android.net.ConnectivityManager
+import androidx.core.content.getSystemService
+import androidx.test.espresso.IdlingResource
+import androidx.test.platform.app.InstrumentationRegistry
+import org.mozilla.fenix.ext.isOnline
+
+class NetworkDisconnectedIdlingResource : IdlingResource {
```

Comment:
If we decide to continue with `IdlingResource`, this change isn't that important because I don't expect the classes to change much but I notice these two classes have a lot of duplication. Is there a way we can share more of their implementation?

Reply:
I removed this class and modified the implementation to work for both